hack/.vendor-helpers.sh

```diff
@@ -118,6 +118,7 @@ clean() {
 
 	echo -n 'pruning unused files, '
 	find vendor -type f -name '*_test.go' -exec rm -v '{}' +
+	find vendor -type f -name '*test_*.go' -exec rm -v '{}' +
```

Comment:
Just browsing on my phone, so ignore me if it's silly; will the leading `*` also be matching, e.g. `latest_version.go`? Could be a problem if it does :smile: